src/skin/popup.css

```diff
@@ -296,6 +302,7 @@ font-size: 16px;
   color: #ccc;
   background: url("/skin/background.png");
   transition: opacity .1s ease;
+  overflow-y: auto;
```

Comment:
What does this fix?

Reply:
A user resizing the textbox to where the buttons (and even the textbox resize handle) are not visible.  Previously, if a user did that, they would have to close the popup in order to reset the textbox size and get the buttons back visible.